README.md

```diff
@@ -245,18 +245,18 @@ These configurations can be loaded from various sources with a deterministic pre
 ## Defaults
 Defaults for AEMM configuration are sourced throughout code. Examples below:
 * **CLI flags**
-  * [server config defaults](https://github.com/aws/amazon-ec2-metadata-mock/blob/master/pkg/config/server.go#L22) 
```

Comment:
Just curious, was the trailing whitespace on these various lines removed by an automated process or did you do it by hand?

I double checked that none of these instances are being treated as significant for the formatting, so this change won't cause any issues.

Reply:
Yeah, my editor does this for me.  Most of the time it's helpful, but for stuff like this it would be easier to either omit totally, or at least put in a separate commit.

Reply:
Sounds good. I'll go ahead and merge it like this.

Reply:
Thanks!